test/e2e/workflow/deployment_test.go

```diff
@@ -143,10 +149,10 @@ var _ = Describe("NetworkAddonsConfig", func() {
 				MultusComponent,
 			}
 			configSpec := cnao.NetworkAddonsConfigSpec{
-				LinuxBridge: &cnao.LinuxBridge{},
-				Multus:      &cnao.Multus{},
-				Ovs:         &cnao.Ovs{},
-				MacvtapCni:  &cnao.MacvtapCni{},
+				LinuxBridge:            &cnao.LinuxBridge{},
```

Comment:
Unrelated to this PR. Please commit format fixes in a separate commit.

Reply:
Done